draft-irtf-cfrg-vdaf.md

```diff
@@ -2340,6 +2380,13 @@ def helper_proof_share(Prio3, agg_id, k_share):
         Prio3.Flp.PROOF_LEN,
     )
 
+def expand_input_share(Prio3, agg_id, input_share):
+    (meas_share, proof_share, k_blind) = input_share
```

Comment:
I think the intent is to map each field in the TLS struct to a variable, but it might be confusing to some people. Is it better to use field name to access each variable. For input share, it's more annoying that you have to condition based on leader or helper here. Will something like this be better?

```
if agg_id > 0:
    # helper, `input_share` is a `Prio3HelperShareWithJointRand`.
    meas_share, proof_share, k_blind = input_share.meas_share, input_share.proof_share, input_share.k_blind
    meas_share = Prio3.helper_meas_share(agg_id, meas_share)
    proof_share = Prio3.helper_proof_share(agg_id, proof_share)
else:
    # leader, `input_share` is a `Prio3LeaderShareWithJointRand`.
    meas_share, proof_share, k_blind = input_share.meas_share, input_share.proof_share, input_share.k_blind
return (meas_share, proof_share, k_blind)
```